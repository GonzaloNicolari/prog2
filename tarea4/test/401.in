# pruebas para pila
es_vacia_pila
es_llena_pila
apilar 1
es_vacia_pila
es_llena_pila
cima
desapilar
es_vacia_pila
es_llena_pila
apilar 1
apilar 2
apilar 3
apilar 4
es_vacia_pila
es_llena_pila
cima
desapilar
cima
desapilar
cima
apilar 3
apilar 4
apilar 5
apilar 6
apilar 7
apilar 8
apilar 8
cima
es_vacia_pila
es_llena_pila
apilar 9
cima
es_vacia_pila
es_llena_pila
cima
desapilar
desapilar
cima
apilar 9
es_llena_pila
apilar 10
es_llena_pila
cima
desapilar
desapilar
desapilar
desapilar
desapilar
desapilar
desapilar
es_vacia_pila
es_llena_pila
cima
desapilar
desapilar
es_vacia_pila
es_llena_pila
cima
desapilar
es_vacia_pila
es_llena_pila
Fin
