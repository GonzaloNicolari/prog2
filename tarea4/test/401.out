1>#  pruebas para pila.
2>p está vacia.
3>p NO está llena.
4>Apilado.
5>p NO está vacia.
6>p NO está llena.
7>1.
8>Desapilado.
9>p está vacia.
10>p NO está llena.
11>Apilado.
12>Apilado.
13>Apilado.
14>Apilado.
15>p NO está vacia.
16>p NO está llena.
17>4.
18>Desapilado.
19>3.
20>Desapilado.
21>2.
22>Apilado.
23>Apilado.
24>Apilado.
25>Apilado.
26>Apilado.
27>Apilado.
28>Apilado.
29>8.
30>p NO está vacia.
31>p NO está llena.
32>Apilado.
33>9.
34>p NO está vacia.
35>p está llena.
36>9.
37>Desapilado.
38>Desapilado.
39>8.
40>Apilado.
41>p NO está llena.
42>Apilado.
43>p está llena.
44>10.
45>Desapilado.
46>Desapilado.
47>Desapilado.
48>Desapilado.
49>Desapilado.
50>Desapilado.
51>Desapilado.
52>p NO está vacia.
53>p NO está llena.
54>3.
55>Desapilado.
56>Desapilado.
57>p NO está vacia.
58>p NO está llena.
59>1.
60>Desapilado.
61>p está vacia.
62>p NO está llena.
63>Fin.
